<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WifiSelection</class>
 <widget class="QDialog" name="WifiSelection">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wifi Network Selection</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>331</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="wifiListWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>331</width>
     <height>221</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string>wifilist</string>
   </property>
   <property name="accessibleDescription">
    <string>List of Wifi Access Points</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passPhraseLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>331</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="passPhraseLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Pass Phrase</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>WifiSelection</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>WifiSelection</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
